src/index.d.ts

```diff
@@ -210,12 +212,12 @@ declare namespace RamdaAdjunct {
         /**
          * Checks if value is a positive `Number` primitive or object.
          */
-        isPositive(val: any): boolean
+        isPositive(val: any): val is number
```

Comment:
Are we sure here ? `isPositive` returns false for the NaN, -Infinity, +Infinity. Nevertheless they are still valid `number` values. The same applies for `isNegative`, `isInteger`, `isFloat`

Reply:
I dont know how to narrow number type to omit named values

stackoverflow too
https://stackoverflow.com/questions/46389560/how-to-narrow-number-type-to-omit-nan-infinity-infinity

Reply:
even haskell cant do this I think :smile: (but idris can)